src/EnrollPasswordController.js

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/ValidationUtil',
+  'util/FactorUtil',
+  'views/enroll-factors/Footer',
+  'views/shared/TextBox'
+],
+function (Okta, FormController, ValidationUtil, FactorUtil, Footer, TextBox) {
```

Comment:
looks like `FactorUtil.js` is not used at all. are changes to `FactorUtil.js` still necessary?

Reply:
good catch, its not used i can remove.

Reply:
if so, you could revert the changes to `FactorUtil.js`?

Reply:
hmmm, that change is still needed to show the description for password factor enroll.